core/server/data/validation/index.js

```diff
@@ -42,6 +43,27 @@ validator.extend('isTimezone', function isTimezone(str) {
     return moment.tz.zone(str) ? true : false;
 });
 
+validator.extend('isEmptyOrWebsite', function isEmptyOrWebsite(str) {
```

Comment:
> Have to agree with @kevinansfield here. It's not good UX to tell users their URL is invalid when it clearly is.

The url shouldn't be invalid. The idea for now was to ignore query params at all. This need's to be changed in the PR if we keep the current solution. @AileenCGN 

@kevinansfield will reply later, need to leave